### DISCUSSION TITLE
Integrate civo kubernetes with Dagger

### DISCUSSION THREAD

**Discussion by saiyam1814** [dagger/dagger#1881] [Ideas]
So after I was able to deploy to Civo Kubernetes (which is K3s based) There was s thought of integration and I am ver well onboard with this.

**Comment by aluzzardi** [comment]
Thank you for the feedback, @saiyam1814

We'll look into Civo. Would you be interesting in contributing the support by any chance?

**Comment by saiyam1814** [comment]
Oh Yes why not? I dpn't know cue tbh but I would want to contribute in any way, even if it required learn by doing I am up for it.

**Comment by saiyam1814** [comment]
So civo has a civo cli that is currently used to manage all resources within civo, what are the next steps for me in order to have civo dagger integration ? 
cc - @shykes

**Comment by shykes** [comment]
Hi! Next step would be to look at equivalent integrations already available, and adapt them to Civo. Then open a work-in-progress PR to get early feedback and additional help.

The closest integration available is probably Google Cloud GKE. See `stdlib/gcp/gke`. Also see the Dagger tutorial in the docs, there is a section on Kubernetes deployment

Thanks for offering to contribute!

**Comment by saiyam1814** [comment]
Yes the Kubernetes deployment works fine on Civo Kubernetes, Let me have a read through some of the PR's to have an idea  and then list down things that needs to be done and also the PR to get early feedback.
